### PR TITLE
Add Albums upload

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ <skjnldsv@protonmail.com>

### CODE REVIEW

File: src/views/AlbumContent.vue [hunk 1]
```diff
@@ -157,71 +152,80 @@
...
-import ImagePlus from 'vue-material-design-icons/ImagePlus'
 import Close from 'vue-material-design-icons/Close'
+import Delete from 'vue-material-design-icons/Delete'
 import Download from 'vue-material-design-icons/Download'
 import DownloadMultiple from 'vue-material-design-icons/DownloadMultiple'
+import ImagePlus from 'vue-material-design-icons/ImagePlus'
+import MapMarker from 'vue-material-design-icons/MapMarker'
+import Pencil from 'vue-material-design-icons/Pencil'
+import Plus from 'vue-material-design-icons/Plus'
+import PlusSvg from '@mdi/svg/svg/plus.svg'
```
Comment (artonge) [inline src/views/AlbumContent.vue:172]: Any reason for reordering ? Maybe we should align to limit those kind of changes

Comment (skjnldsv) [reply src/views/AlbumContent.vue:172]: Cleaning messy code :)

**Pull request merged**